HASHCUT69/Blend-n-Run
# NEW FEATURE - User Review Prompt

### **Pull Request Description:**

**New Feature:** User Review Prompt

**Description:**
- ### **Objective:** Enhance user engagement and feedback for the extension.
- ### **User Interaction:**
  - After every 10 uses, users will receive a non-intrusive popup.
  - Options include: **Review**, **Ignore**, and **Don't Ask Again**.
- ### **Behavior:**
  - If **Ignored**: The prompt repeats after the next 10 uses.
  - If **Don't Ask Again**: The prompt never shows up.
  - If **Review**: The prompt never shows up again.
- ### **Implementation:**
  - Utilizes local storage to track user interactions.
  - Stylish modal layout for a visually appealing experience.

### **Testing Steps:**
1. Use the extension for 10 times.
2. Observe the appearance of the review prompt.
3. Test each option: *Review*, *Ignore*, and *Don't Ask Again*.
4. Verify the correct behavior based on the chosen option.

### **Notes for Reviewers:**
- Ensure the modal layout is visually consistent.
- Validate that user choices persist across sessions.
- Confirm the prompt does not interfere with the extension's primary functionality.

### **Additional Information:**
- Linked to relevant issues or user stories.
- Responsive design considerations applied for varied screen sizes.
- Follows established coding standards and best practices.
